components/developer/vala/Makefile

```diff
@@ -52,6 +56,10 @@ build:      $(BUILD_32_and_64)
 
 install:    $(INSTALL_32_and_64)
 
+# Failed tests: /methods/bug653391 - cc1: some warnings being treated as errors
```

Comment:
Better put test results in test/results.master, set COMPONENT_TEST_RESULTS and use COMPONENT_TEST_TRANSFORMS to make them reproducible.